boba-chain-ops/Makefile

```diff
@@ -20,3 +21,6 @@ boba-rollover:
 
 boba-crawler:
 	env GO111MODULE=on go build -o ./bin/boba-crawler ./cmd/boba-crawler
+
+boba-devnet:
+	env GO111MODULE=on go build -o ./bin/boba-devnet ./cmd/boba-devnet
```

Comment:
I realize this is copy-pasted, but, it's a little odd to explicitly set `GO111MODULE=on` as this has been the default since go 1.16

Reply:
Removed